### DISCUSSION TITLE
Remove the border on the viewer frame

### DISCUSSION THREAD

**Discussion by wmkoolen** [bordaigorl/rmview#52] [General]
Dear author,
This is a useful package, thank you. I found the border around the main viewing panel look out of place, especially when operating the application fullscreen. This patch fixed it for me:

```
diff --git a/src/rmview/viewer.py b/src/rmview/viewer.py
index f235542..b50c1da 100644
--- a/src/rmview/viewer.py
+++ b/src/rmview/viewer.py
@@ -1,6 +1,6 @@
 from PyQt5.QtCore import Qt, QRectF, pyqtSignal, QT_VERSION_STR
 from PyQt5.QtGui import QWindow, QImage, QPixmap, QTransform, QIcon
-from PyQt5.QtWidgets import QApplication, QGraphicsView, QGraphicsScene, QFileDialog, QAction, QMenu
+from PyQt5.QtWidgets import QApplication, QGraphicsView, QGraphicsScene, QFileDialog, QAction, QMenu, QFrame
 
 
 class QtImageViewer(QGraphicsView):
@@ -12,6 +12,8 @@ class QtImageViewer(QGraphicsView):
     QGraphicsView.__init__(self)
     # self.setAttribute(Qt.WA_OpaquePaintEvent, True)
 
+    self.setFrameStyle(QFrame.NoFrame)
+
     self.scene = QGraphicsScene()
     self.setScene(self.scene)
```

**Comment by bordaigorl** [comment]
Which platform are you on?
I really don't see any difference on my Mac...

**Comment by wmkoolen** [comment]
![with](https://user-images.githubusercontent.com/29552933/109890646-f05d2f00-7c87-11eb-9ae8-b9feb5cd6088.png)
![without](https://user-images.githubusercontent.com/29552933/109890650-f0f5c580-7c87-11eb-8140-879bb412f715.png)

Dear bordaigorl,

I'm using KDE 5.20 on Debian, with breeze as the plasma style.

Here are two screenshots of the top-left part of your application window.

You can see in 'with.png' that there is a border consisting of a gray 
and a blue line inserted around the view element. With the update I sent 
you, we're getting 'without', where that border is gone.

Cheers,

Wouter

**Reply by bordaigorl** [reply]
I see! Thanks for the screenshots. I will incorporate this change in the next release
